app/(app)/town-lines/page.tsx

```diff
@@ -0,0 +1,69 @@
+"use client";
+
+import { useRef } from "react";
+import Map, { Layer, MapRef, Source } from "react-map-gl";
+import intersections from "../../db/geos/intersections.json";
+import townLines from "../../db/geos/town_line_poly.json";
+
+export default function TownLinesPage() {
+  const mapRef = useRef<MapRef>(null);
+
+  return (
+    <div className="flex flex-row justify-center gap-10 px-5 py-14 md:px-[20%] md:py-20">
+      <div className="flex flex-col items-center justify-center gap-7">
```

Comment:
```suggestion
      <div className="flex w-full flex-col items-center justify-center gap-7">
```

Reply:
This will fix a bug where the map and text get truncated when resizing the window from large to smaller screen size on desktop